Automattic/wp-calypso
# Cancel Purchase: Make refund currency more accurate

## Proposed Changes

The cancel purchases page uses `formatCurrency()` to format its refund price, however, it assumes that the refund amount is in the same currency as the purchase's currency. This can be inaccurate.

In this diff we modify the displayed price to use the refund amount's currency instead.

Requires D103872-code

Fixes https://github.com/Automattic/payments-shilling/issues/1411

Before:

<img width="956" alt="Screenshot 2023-03-07 at 2 54 57 PM" src="https://user-images.githubusercontent.com/2036909/223538036-0a8e8595-d0f2-4547-8752-ae0261bc428e.png">

After:

<img width="964" alt="Screenshot 2023-03-07 at 2 51 26 PM" src="https://user-images.githubusercontent.com/2036909/223538060-ba18bf75-c25e-4ef9-b28b-b1bd72f63c9b.png">


## Testing Instructions

- You'll need a recent, refundable purchase in calypso. If you do not have one, make a purchase of a plan.
- Switch your user's account to a different currency using Store Admin.
- Visit the `/me/purchases` page and click on that purchase.
- Click the "Cancel purchase" button at the bottom.
- Verify that you see the amount to be refunded displayed at the bottom of the page.
- Verify that the refunded amount's currency symbol matches the ORIGINAL currency your purchase was made in, not the current currency.
